### PR TITLE
Added ls feature to show.py

### PR DESCRIPTION
**Pull request by debabrata-15**
Added the ls feature just to show.py just have to paste the stash directory

### CODE REVIEW

File: package/show.py [hunk 1]
```diff
@@ -66,4 +66,13 @@ def copy_to_clipboard(text):
             check=True)
 
     else:
-        raise OSError("Unsupported operating system")
\ No newline at end of file
+        raise OSError("Unsupported operating system")
+def display(*args):
+    if not args:
+        ls('package/stash') #Enter stash directory
```
Comment (this-is-yaash) [inline package/show.py:72]:
```suggestion
        ls('package/stash') 
```

**Review by this-is-yaash — COMMENTED** [review]
This method works perfectly, but it prevents me from using the existing method such as **print code in output** and **clipboard copy**.

Try to execute the existing methods we have, by creating a `test.py` file in `package/` directory.

```py
#test.py
from show import display
display("test", 1007) #current time = 10:07 -> 1007
display("test", 1007, 1) # this method copies the content into system's clipboard
```

**Comment by debabrata-15** [comment]
I have changed my implementation a bit this one works better

**Review by this-is-yaash — APPROVED** [review]
LGTM!

**Comment by this-is-yaash** [comment]
@debabrata-15 We'll merge it **ASAP**, once you **resolve the merge conflicts**!

**Comment by debabrata-15** [comment]
> @debabrata-15 We'll merge it **ASAP**, once you **resolve the merge conflicts**!

I have resolved the merge conflicts

File: package/show.py
Comment (ShashwatAgrawal20) [inline package/show.py:1]: you still need to format the code.

**Review by ShashwatAgrawal20 — APPROVED** [review]

File: package/show.py
Comment (debabrata-15) [reply package/show.py:1]: do you mean writing comments for the changes I made?

Comment (ShashwatAgrawal20) [reply package/show.py:1]: No, I meant formatting the code using a code formatter. 

maybe if you're on vscode the option will be somewhere after you right click, not sure tho. You can also use `black` for that.

**Comment by this-is-yaash** [comment]
@debabrata-15 merging the PR.
thank you for your contributions!

**Comment by debabrata-15** [comment]
> @debabrata-15 merging the PR.
> thank you for your contributions!

Thank you for giving me this opportunity and being patient with me

**Comment by ShashwatAgrawal20** [comment]
@debabrata-15 dude, make sure to use a proper code formatted like black this code wasn't formatted.

**Comment by debabrata-15** [comment]
I did use black to format the code 
I'll do that once again

**Pull request merged**